extensions/wikia/AdEngine/js/PageFairDetection.js

```diff
@@ -0,0 +1,71 @@
+define('ext.wikia.adEngine.pageFairDetection', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.scriptLoader',
+	'wikia.browserDetect',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, scriptLoader, browserDetect, doc, log, win) {
+	'use strict';
+
+	var context = null,
+		logGroup = 'ext.wikia.adEngine.pageFairDetection';
+
+	function getContext() {
+		if (context === null) {
+			context = adContext.getContext();
+		}
+		return context;
+	}
+
+	function skinIsMobile() {
+		return getContext().targeting.skin === 'mercury';
+	}
+
+	function getWebsiteCode() {
+		var websiteKeys = {
+			mobile: 'FA8D90F8D9F54665',
+			desktop: 'FE3882548B7E471A'
+		};
+
+		return skinIsMobile() || browserDetect.isMobile() ? websiteKeys.mobile : websiteKeys.desktop;
+	}
+
+	function dispatchDetectionEvent(eventName){
```

Comment:
It's going to be refactored soon :yum:
https://github.com/Wikia/app/pull/10901/files#diff-c83f41e55c500a05e9e8087d515f1643R1